### DISCUSSION TITLE
[ANAC2024] My agent is listed on "Missing Metadata" area, even if I have provided metadata

### DISCUSSION THREAD

**Discussion by a4rcvv** [yasserfarouk/anl#2] [Q&A] [answered]
(sorry if it is not suitable to ask about ANAC competition at GitHub Discussions)

I found that my agent (AgentNyan) is listed on "Missing Metadata" area of ANL Leaderboard after code submission was closed.
I think I have provided metadata (because without metadata, a test run cannot be performed).
Also, I think my agent was on "Running in the official compeition" area at 4/15 19:00.

So could you check a status of my submission?

![image](https://github.com/yasserfarouk/anl/assets/35623957/6c45e65d-94f7-4832-8325-a95adfff5455)

**Comment by yasserfarouk** [comment, marked as answer]
Dear Ryota Arakawa

This was fixed. Your agent is running normally in the qualifications and
was not interrupted

Thanks
yasser
